packages/build-tools/src/try-collect-imports-deep.ts

```diff
@@ -0,0 +1,18 @@
+import type { Stylable, StylableMeta } from '@stylable/core';
+
+export function tryCollectImportsDeep(
```

Comment:
seems like this could be a part of `core`, probably in a helper/import with `visitMetaCSSDependencies`

Reply:
maybe in `v5` this should be handled via `stylable.getDependencies`, adding resiliency and error reporting